lib/cli-engine.js

```diff
@@ -453,6 +512,13 @@ CLIEngine.prototype = {
 
         stats = calculateStatsPerRun(results);
 
+        if (options.useCache) {
+            // persist the cache to disk
+            fileCache.reconcile();
+        }
+
+        debug("Time: " + (Date.now() - startTime) + "ms");
+
```

Comment:
I think this is the total amount of time it took to lint? If so, maybe "Linting complete in: "